NEW_README.md

```diff
@@ -188,29 +188,43 @@ bundle exec rake opsworks_performance    # (takes about 1.5 hours)
 
 ## Soak performance tests
 
-In order to execute a Soak performance run:
+The soak performance test executes a long-running scenario under medium load:
+* 14 days
+* 600 agents
+* `role::by_size::large`
 
-* Follow the instructions from
-[Just set up PE and Gatling](#set-up-puppet-enterprise-and-gatling-but-do-not-execute-a-gatling-scenario).
+### Provision, tune, run
 
-* Then manually tune the master by running `puppet infrastructure tune` on the master.
+A set of 'soak' rake tasks are provided to handle setup and test execution, allowing nodes to be provisioned as part of the run or as a separate step.
+The pre-suite includes tuning of the master via 'puppet infrastructure tune' so this is no longer a manual step.
 
-* You will have to edit '/etc/puppetlabs/puppet/hiera.yaml' to add 'nodes/%{fqdn}' to the top of the hiera hierarchy.
+Note that when using the soak rake tasks the `BEAKER_PRESERVE_HOSTS` environment variable is always set to 'true', so you will need to de-provision the test nodes with the `performance_deprovision_with_abs` when your testing is complete.
 
-* Then create '/etc/puppetlabs/code/environments/production/hieradata/nodes/\<master fqdn>.yaml' where master fqdn is the result of `facter networking.fqdn` (run on master), and put the output of the tune command in it.
+To execute a soak performance run:
```

Comment:
This line is a repeat of the next one.  Should be deleted.

Reply:
Removed